main-site/components/covid-section/CovidSection.animations.tsx

```diff
@@ -0,0 +1,53 @@
+export const smallPuff = {
```

Comment:
Is there a reason this file extension is tsx? If not lets keep it .ts like the other animation files